Airports/EGLL/VRPs.txt

```diff
@@ -0,0 +1,10 @@
+*Ascot:N051.25.00.570:W000.39.33.070
+*Bagshot:N051.20.45.000:W000.42.30.000
```

Comment:
Bagshot
512057N 0004157W

Reply:
Yeah I move it (and Ascot & Burnham) slightly for clarity against the other airspace features

Reply:
I see now, cheers! I would note that the VRP labels are already a bit deceiving as to the actual position of the VRPs - the default positioning is 'Center Center', so it's actually the middle of each label that is on top of the AIP coordinate. I think it would make sense if we started to change this to 'Left Top', so the * is over the coordinate. But if we are to offset slightly based on the current 'Center Center' positioning, I'd suggest maybe putting a ;Comment at the end of each line that differs from the AIP, especially if we might use this data in [uk-controller-plugin #173](https://github.com/VATSIM-UK/uk-controller-plugin/issues/173)

Reply:
I'll have to double check what I've got it set to in my settings but I'm pretty sure it's set to 'center center.'
Excellent point about UKCP using the data. I don't actually use the free text displays for VRPs much so would be equally happy with either correcting or amending as it won't affect me 😅 
Any strong preference as to which I do?

Reply:
@cbyworth Let’s just leave a comment where they’re ‘off’ for now - and we can all have a chat soon about the future of free text positioning! 😄 Riveting stuff....